packages/core/src/Stores/client-store.js

```diff
@@ -1090,17 +1089,14 @@ export default class ClientStore extends BaseStore {
             await WS.forgetAll('balance');
             await BinarySocket.authorize(this.getToken());
         }
-
+        if (this.root_store.common.has_error) this.root_store.common.setError(false, null);
         sessionStorage.setItem('active_tab', '1');
 
         // set local storage
         this.root_store.gtm.setLoginFlag();
 
         await this.init();
-
-        // broadcastAccountChange is already called after new connection is authorized
-        if (!should_switch_socket_connection) this.broadcastAccountChange();
-
+        this.broadcastAccountChange();
```

Comment:
According to the removed comment, the broadcast was already made, would you please elaborate on why we need to send this again?

Reply:
Unfortunately it's not. This method `broadcastAccountChangeAfterAuthorize` is never called.